content/guides/dapps/journal.md

```diff
@@ -0,0 +1,582 @@
+---
+date: March 18, 2024
+difficulty: intro
+title: "How to create a journal dApp on Solana"
```

Comment:
Imo the title should be something about creating a crud app instead of the journal. Journal is the vehicle for teaching, but the crud app is the focus. This would help with SEO as well

Reply:
Good point. Let's update it to "How to create a CRUD dApp on Solana"